alerting/src/main/kotlin/org/opensearch/alerting/model/Monitor.kt

```diff
@@ -77,6 +79,22 @@ data class Monitor(
                     require(trigger is DocumentLevelTrigger) { "Incompatible trigger [${trigger.id}] for monitor type [$monitorType]" }
             }
         }
+
+        // validate the number of Actions Across Triggers
```

Comment:
We will actually need to do this validation from the `TransportIndexMonitorAction` and check the monitor there since we can get the max action settings in that class. Here, `MonitorRunnerService` wont be instantiated with the values, so this would have errors